Gemfile

```diff
@@ -25,6 +25,7 @@ group :development, :test do
   gem 'rspec-rails', '<= 3.7.2'
   gem 'timecop', '<= 0.9.1'
   gem 'webmock', '<= 3.4.1'
+  gem 'rails-controller-testing'
```

Comment:
Why is this needed? Isn't rsepc-rails enough?

Reply:
@tagnotfound I need to test the controller is render the right template. https://github.com/rails/rails-controller-testing